notebooks/122-quantizing-model-with-accuracy-control/README.md

```diff
@@ -0,0 +1,26 @@
+# Quantization with accuracy control using NNCF
+
+This tutorial demonstrates how to apply `INT8` quantization with accuracy control to the speech recognition model and to PyTorch YOLOv8 model.
+
+Supported models:
+* `122-speech-recognition-quantization-wav2vec2.ipynb` demonstrates how to apply post-training `INT8` quantization with accuracy control on a fine-tuned [Wav2Vec2-Base-960h](https://huggingface.co/facebook/wav2vec2-base-960h) [PyTorch](https://pytorch.org/) model, trained on the [LibriSpeech ASR corpus](https://www.openslr.org/12).
+* `122-yolov8-quantization-with-accuracy-control.ipynb` demonstrates how to apply post-training `INT8` quantization on the [YOLOv8](https://github.com/ultralytics/) PyTorch model
+
+The code of the tutorials is designed to be extendable to custom models and datasets.
+
+The advanced quantization flow allows to apply 8-bit quantization to the model with control of accuracy metric. This is achieved by keeping the most impactful operations within the model in the original precision. The flow is based on the [Basic 8-bit quantization](https://docs.openvino.ai/2023.0/basic_quantization_flow.html) and has the following differences:
```

Comment:
I believe it is based on https://docs.openvino.ai/2023.0/quantization_w_accuracy_control.html. Please fix.

Reply:
Fixed